### PR TITLE
Mapsme 5966 discovery activity

### PR DESCRIPTION
**Pull request by alexzatsepin**
https://jira.mail.ru/browse/MAPSME-5966

### CODE REVIEW

**Comment by greshilov** [comment]
JTTP

File: android/src/com/mapswithme/maps/discovery/DiscoveryFragment.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.mapswithme.maps.R;
+import com.mapswithme.maps.base.BaseMwmToolbarFragment;
+
+
```
Comment (milchakov) [inline android/src/com/mapswithme/maps/discovery/DiscoveryFragment.java]: лишняя строка

**Comment by greshilov** [comment]
JTTP

**Comment by mapsmetest** [comment]
Jenkins asks: can anyone have a look at this request?
If you want Jenkins to compile and test this, write "test this please" or JT_TP (without the underscore in the middle).

**Comment by greshilov** [comment]
JTTP

**Comment by greshilov** [comment]
test this please

**Comment by alexzatsepin** [comment]
@greshilov what do you mean?

**Comment by greshilov** [comment]
@alexzatsepin dont't care about this comments, it's for Jenkins!

File: android/src/com/mapswithme/maps/discovery/DiscoveryFragment.java (hunk 1 above)
Comment (alexzatsepin) [reply android/src/com/mapswithme/maps/discovery/DiscoveryFragment.java]: Fixed

**Pull request merged**